src/test/java/com/google/jenkins/plugins/computeengine/InstanceConfigurationTest.java

```diff
@@ -182,70 +186,68 @@ public void testConfigRoundtrip() throws Exception {
 
   @Test
   public void testInstanceModel() throws Exception {
-    Instance i = instanceConfiguration(MIN_CPU_PLATFORM).instance();
+    Instance instance = instanceConfiguration(MIN_CPU_PLATFORM).instance();
     // General
-    assert (i.getName().startsWith(NAME_PREFIX));
-    assert (i.getDescription().equals(CONFIG_DESC));
-    assert (i.getZone().equals(ZONE));
-    assert (i.getMachineType().equals(MACHINE_TYPE));
-    assert (i.getMinCpuPlatform().equals(MIN_CPU_PLATFORM));
+    assertTrue(instance.getName().startsWith(NAME_PREFIX));
+    assertEquals(instance.getDescription(), CONFIG_DESC);
```

Comment:
Very much a nit: For assertEquals the first parameter should be expected, and second should be actual.

Reply:
Fixed.